### ISSUE TITLE
parameter storing

### ISSUE THREAD

**Issue by TobiasRoeddiger** [edge-ml/edge-ml#18]
why do we store the confusion matrix and classification report as strings?

**Comment by KtrauM** [comment]
There is not a good reason to store them as strings, but it was returned by sklearn in a hard to parse format and nevertheless could be displayed directly. It was a temporary solution that needs to be fixed.

**Comment by TobiasRoeddiger** [comment]
I see why it might make sense to store it as a string because the format depends on the number of classes etc?
